modules/tinymce/src/themes/silver/main/ts/Render.ts

```diff
@@ -68,6 +68,9 @@ export interface RenderArgs {
   readonly height: string;
 }
 
+const SpapMessage = 'Try Premium!';
```

Comment:
Can we please move all this "SPAP" logic out into a separate module? It just makes this messy. Ideally it'd also not be called SPAP as that is not clear what it means... I'd suggest renaming all this to just be `promotion` instead.

To clarify, I mean have a separate module that has a `render` function or similar that returns the promotion `AlloySpec` which is then used in the `OuterContainer.parts.spap` call below.

Reply:
Also these names aren't camel case when they should be.